### DISCUSSION TITLE
Home accounts chart null data

### DISCUSSION THREAD

**Discussion by stuzer05** [firefly-iii/firefly-iii#8112] [Firefly III] [answered]
So when I show "Everything" on home page accounts chart for some reason I see a whole year of basically nothing in the beginning. How to get rid of that empty data in the chart?
![image](https://github.com/firefly-iii/firefly-iii/assets/23124682/38693c22-713c-43cf-8a51-3330934c6ff3)

**Comment by JC5** [comment]
Your oldest transaction is at that moment.

**Reply by stuzer05** [reply]
The oldest transaction is almost a year ahead 
![image](https://github.com/firefly-iii/firefly-iii/assets/23124682/c67f1ea4-a679-4a45-b032-e2dcd2fcce1f)

**Reply by JC5** [reply]
Check the date field in the transaction_journals table or simply scroll back in your accounts.

**Reply by stuzer05** [reply]
Yeah, there're dates like that. But it seems like SQL doesn't ignore `deleted_at` column. Because nothing changed in the chart
![image](https://github.com/firefly-iii/firefly-iii/assets/23124682/8a5750f6-a942-4c06-bacf-07bee9109f10)

**Reply by JC5** [reply, marked as answer]
Browse to /flush first.

**Reply by stuzer05** [reply]
Now it worked, thank you!
